src/App/Transfers/Dashboard/ErrorsByPayeeChart.tsx

```diff
@@ -36,7 +37,7 @@ const ByPayeeChart: FC<ConnectorProps> = ({ filtersModel, onFilterChange }) => {
 
   const [activeIndex, setActiveIndex] = useState<number>();
 
-  const onPieEnter = (_: any, index: number) => {
+  const onPieEnter = (pieData: any, index: number) => {
```

Comment:
Why modify this, out of curiosity? Should it be
```suggestion
  const onPieEnter = (_pieData: any, index: number) => {
```
to indicate that it's unused, but still communicate to the user what it is?

Reply:
Because I think it's standard for `lodash` to be imported as `_`.

but now that I think of it...it's probably better to import specific methods instead of importing the whole library.

And yes `_pieData` would be best for the user so they have an idea what it is without looking up `recharts` docs.